decidim-core/lib/decidim/query_extensions.rb

```diff
@@ -15,7 +15,7 @@ def self.extend!(type)
           type !types[ProcessType]
           description "Lists all processes."
 
-          resolve ->(_obj, _args, ctx) {
+          resolve lambda { |_obj, _args, ctx|
```

Comment:
Yeah, I'm not sure I like the lambda style better anyways... My preferred style is to always use arrows but using `do...end` for multilines.